pymongo/monitoring.py

```diff
@@ -1281,7 +1281,7 @@ def __repr__(self) -> str:
         )
 
 
-class TopologyOpenedEvent(TopologyEvent):
+class TopologyOpeningEvent(TopologyEvent):
```

Comment:
This is a breaking change so we have to live with the "TopologyOpenedEvent" name.

Reply:
We already do that here: https://github.com/mongodb/mongo-python-driver/blob/272538513a589d8bf795ba0b7591f37414e96919/test/unified_format.py#L917-L919C23